### ISSUE TITLE
Sketcher: Ability to view constraints automatically

### ISSUE THREAD

**Issue by Roberto8303** [FreeCAD/FreeCAD#17361]
### Original PR

Freecad

### Description and Justification

It would be useful to be able to automatically display once a constraint is selected on the drawing in the constraints table. Now you have to scroll and look for it among everyone and it takes more time.  
![image](https://github.com/user-attachments/assets/c5fab8ff-bb4f-4f8c-ac3b-2442bb7fe82d)
![Uploading image.jpg…]()

**Comment by Roberto8303** [comment]
they should light up highlighting all the selected ones.
It happens when there are a lot of constraints in a sketch and if you want to hide someone in particular because it disturbs the view of the sketch, pressing the space bar does not hide it and you have to search for it in the constraints list.
so if it would be highlighted automatically you could immediately disable the view with the check mark.

**Comment by maxwxyz** [comment]
so the selected constraints is already highlighted in the 3D view and the list. What version are you using, please post full about info.

**Comment by Roberto8303** [comment]
they are already highlighted but you have to scroll to find them and see them highlighted. It would be nice if the bar automatically moved to the foreground to show the selected ones.
the version is 1.1dev

**Comment by Beep6581** [comment]
> So it should scroll to the selected constraint? What if multiple are selected? related #5877

Yes, it should scroll to the selected constraint item in the QListView. If multiple constraints are selected, scroll to the last-selected one.
